### PR TITLE
Heavy gauss rifle

### PR DESCRIPTION
**Pull request by Alasnuyo**
Well, I've removed the burst fire mechanism and slashed the ammo capacity by 90 percent. Let's see how this goes.

### CODE REVIEW

File: data/json/items/ranged.json [hunk 1]
```diff
@@ -4762,6 +4762,36 @@
...
+	"ranged_damage" : 40,
+	"range" : 40,
+	"dispersion": 100,
+	"sight_dispersion": 150,
+	"aim_speed" : 6,
+	"recoil": 300,
+	"durability": 7,
+	"clip_size": 10,
+	"reload": 300,
+	"valid_mod_locations": [["accessories", 4],["barrel", 1],["grip", 1],["mechanism", 4],["muzzle", 1],["rail", 1],["sights", 1],["stock", 1],["underbarrel", 1]]
```
Comment (KA101) [inline data/json/items/ranged.json]: I'm still not sold on mounting a muzzle brake or an extended barrel on a coilgun.

Comment (kevingranade) [inline data/json/items/ranged.json]: Agreed, nothing that interacts with the projectile makes any sense, if you
wanted to make mods that affect coilgun projectiles, it needs a new mod
slot.

**Comment by Alasnuyo** [comment]
Right. Barrel and muzzle mod-slot removed. How does it look?

File: data/json/items/ranged.json [hunk 2]
```diff
@@ -4762,6 +4762,36 @@
...
+	"ranged_damage" : 40,
+	"range" : 40,
+	"dispersion": 100,
+	"sight_dispersion": 150,
+	"aim_speed" : 6,
+	"recoil": 300,
+	"durability": 7,
+	"clip_size": 10,
+	"reload": 300,
+	"valid_mod_locations": [["accessories", 4],["grip", 1],["mechanism", 4],["rail", 1],["sights", 1],["stock", 1],["underbarrel", 1]]
```
Comment (kevingranade) [inline data/json/items/ranged.json]: Mechanism is problematic too, it includes things like "rapid blowback", "auto-fire mechanism", "match trigger".

File: data/json/recipes/recipe_weapon.json [hunk 3]
```diff
@@ -5634,6 +5634,112 @@
...
+	"category": "CC_WEAPON",
+	"subcategory": "CSC_WEAPON_RANGED",
+	"skill_used": "fabrication",
+	"skills_required": [["mechanics", 10], ["electronics", 9], ["computer", 2]],
+	"difficulty": 11,
+	"time": 720000,
+	"reversible": false,
+	"autolearn": true,
+	"qualities":
+		[
```
Comment (kevingranade) [inline data/json/recipes/recipe_weapon.json]: Your indention is pretty wacky, I suspect your editor is inserting tabs or something.

**Comment by Alasnuyo** [comment]
Okay. Mechanism removed and indentation corrected.

**Comment by KA101** [comment]
OK, this merge-conflicted with the water cannon because both guns had the classic "oh I'll just stick it at the end of the list" issue.  I moved this underneath the existing rail rifle, and in so doing found and am taking the liberty of fixing a few issues.
1) Loading 10 rails into the unit at a time should not take (only) 100 more moves than loading one rail into the standard rifle.  To that end, I gave this the RELOAD_ONE tag and presume that one makes a manual loader out of the various electronics and metal in the recipe.
2) Changed the id/name to "heavy rail rifle" to reduce confusion, as this is essentially the rail rifle with a magazine, accessory mounts, and considerably more reach & oomph.

**Comment by Alasnuyo** [comment]
In my head, I imagined that thing to have a gravity fed magazine, hence the speed it's loading. But if you think your way less immersion breaking, I have no objection to that.

**Comment by KA101** [comment]
Top-mount mag would preclude a scope or effective rail mounting, I'd think.  Sorry I didn't see that earlier.

**Pull request merged**